GreenGarbageApp/app/src/main/res/layout/fragment_review.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:paddingTop="180dp"
+    tools:context="com.example.greengarbageapp.activities.MainActivity">
+
+    <TextView
+
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:padding="8dp"
+
+        android:fontFamily="@font/acme"
+        android:textColor="@color/green_dark"
+        android:gravity="center"
+        android:text="@string/help_us_improve_the_app"
+        android:textSize="35sp"/>
+    <Space
```

Comment:
Space is one way to go, I prefer padding, since it just one line, and if you change the layout, the padding on the textView or ImageView would keep its padding. I could also be specific like paddingTop, padding Start, padding End and paddingBottom.

Like this: ( I use this ALOT 😅)
android:padding="20dp"